src/api/Annotations.js

```diff
@@ -138,11 +172,14 @@ export default class Annotations extends MarkerBasedApi {
             return;
         }
 
+        const requestData = shouldFetchReplies ? { fields: 'replies' } : null;
```

Comment:
Does this need to be `null`? We should generally only use `null` values if there is a difference between it and `undefined`.